cypress/integration/form.spec.ts

```diff
@@ -0,0 +1,70 @@
+// const book_title = 'The Little Schemer';
+
+describe('Form Actions', () => {
+  beforeEach(() => {
+    cy.visit('/#/examples/form');
+  });
+
+  it('should be able to input all values', () => {
+    cy.get('mat-slide-toggle[formcontrolname="autosave"]').click();
+    cy.get('mat-form-field input[formcontrolname="username"]').type(
+      'someusername'
+    );
+    cy.get('input[formcontrolname="password"]').type('somepassword');
+    cy.get('input[formcontrolname="email"]').type('some@email.com');
+    cy.get('input[formcontrolname="birthday"]').type('1/1/2000');
+    cy.get('textarea[formcontrolname="description"]').type('some description');
+    cy.get('mat-checkbox[formcontrolname="requestGift"] label').click();
+
+    // TODO: slider
+    cy.get('mat-slider[formcontrolname="rating"]').click();
```

Comment:
I think we should also click the `send` action here and we could test if the snackbar shows up.
Thoughts?

Reply:
Added code to test the snackbar notifications. Did not even notice them before!